exercises/js/inputHint.js

```diff
@@ -0,0 +1,50 @@
+class Hint {
+  constructor(options) {
+    this.parentForm = $(options.formSelector);
+    this.inputField = this.parentForm.find(options.inputSelector);
+    this.labelField = this.parentForm.find(options.labelSelector);
+  }
+
+  init() {
+    this.removeLabelFeild();
+    this.bindEventHandlers();
+  }
+
+  removeLabelFeild() {
+    // Set the value of the search input to the text of the label element
+    let labelText = this.labelField.text();
+    this.inputField.data("labelText", labelText);
+    this.inputField.val(labelText);
```

Comment:
This can be extracted in a method as I said in previous changes.
You can extract method to set value of input field and use it everywhere.